Pages/AboutUs.razor

```diff
@@ -2,6 +2,38 @@
 
 <h3>AboutUs</h3>
 
-@code {
+<div class="team-container">
+    <div class="team-member">
+        <img src="path-to-image1.jpg" alt="Member 1">
+        <p>Member 1</p>
+    </div>
+    <div class="team-member">
+        <img src="path-to-image2.jpg" alt="Member 2">
+        <p>Member 2</p>
+    </div>
+    <div class="team-member">
+        <img src="path-to-image3.jpg" alt="Member 3">
+        <p>Member 3</p>
+    </div>
+</div>
+<div class="company-info">
+    <p>
+        Welcome to Ethcial Checkers, your dedicated platform for exploring the ethical dimensions of everyday questions through AI.
 
-}
+        We are a trio of Computer Science students from Southern Methodist University (SMU) with a passion for leveraging artificial intelligence to foster thoughtful discussions and ethical awareness. Our journey began in the classroom, where shared interests in AI's potential and ethical applications brought us together.
+
+        Our project, Ethical Checkers, was born out of a desire to see how different people might respond to various ethical questions, ranging from everyday dilemmas to complex moral quandaries. By utilizing advanced AI algorithms, we aim to provide a unique platform where users can not only see various ethical perspectives but also gain insight into the reasoning behind them.
+
+        Our Team:
+        <br /><br />
+        Quinn specializes in machine learning and AI ethics, ensuring our algorithms adhere to the highest ethical standards.
+        <hr />
+        Kevin focuses on user experience and interface design, making Ehtical Chekers as user-friendly and engaging as possible.
+        <hr />
+        Mclean leads our backend development, ensuring robust and scalable infrastructure to handle queries efficiently.
+        <br /><br />
+        At Ethical Chekers, we believe in the power of technology to enhance our understanding of ethics in the digital age. We invite you to join us in this exciting exploration, pose your questions, and discover a multitude of ethical viewpoints.
+
+        Stay curious and engaged—because at Ethical Checkers, every question is an opportunity to learn and grow ethically. should know about your company.
```

Comment:
```suggestion
        Stay curious and engaged—because at Ethical Checkers, every question is an opportunity to learn and grow ethically.
```

Typo/extraneous text